spec/firebase_app_distribution_action_spec.rb

```diff
@@ -1,141 +1,147 @@
 describe Fastlane::Actions::FirebaseAppDistributionAction do
   let(:fake_file) { StringIO.new }
-
-  before do
+  let(:temp_connection) { double("Connection") }
+  let(:fake_get_app_response_valid) { double("Response", status: 200, body: { projectNumber: "1", appId: "1", platform: "android", bundleId: "1", contactEmail: "Hello@world.com" }) }
```

Comment:
I can't entirely tell from the diff where all these `let` variables get used, but consider inlining them in their corresponding test case if they are only used once.  It's easier to read specs when the setup is closer to the verification and can help you avoid some testing antipatterns like `The Mystery Guest` and `General Fixture` helpful ref: https://thoughtbot.com/blog/lets-not